File: skills/typescript-coding/data/fixtures/violations.ts
// Intentional violations — exactly one per checker rule — used by the tests.
export function demo(config: unknown): void {
  console.log('starting');
  const url = process.env.DATABASE_URL;
  const status: any = url;
  const value = status!.trim();
  void value;
}

enum Color { Red, Green }

// @ts-ignore
export const broken = demo;

it.only('focused', () => undefined);

File: skills/python-coding/data/fixtures/violations.py
# Intentional violations — exactly one per checker rule — used by the tests.
def demo(config: object) -> None:
    print("starting")
    url = os.environ.get("DATABASE_URL")
    value: Any = url
    try:
        handle(value)
    except:
        pass
    assert config is not None
    breakpoint()


# type: ignore
broken = demo
